### PR TITLE
Task 1028

### PR DESCRIPTION
**Pull request by tcavaletto**
Task 1028, 1029 done.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
-  before_filter :configure_permitted_parameters, if: :devise_controller?
```
Comment (bammons) [inline app/controllers/application_controller.rb:7]: This needs to be before_filter. rubocop is just being dumb

**Comment by bammons** [comment]
Too much of the same function repeated over and over again. This really needs to be cleaned up

**Comment by bammons** [comment]
:+1:

**Comment by seth-marquez** [comment]
+1 looks good
heavy

**Comment by Linkster700** [comment]
+1

**Comment by bammons** [comment]
still :+1:

**Pull request merged**